DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/SetLearningLogItemCompletionDateViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+{
+    public class SetLearningLogItemCompletionDateViewModel : SetLearningLogItemCompletionDateFormData

```

Comment:
Minor, but it would be good to have consistent naming over the new action/model/view/route names. 

I'm debating as to whether "Set..." or "Mark..." is the better choice here. I think I am leaning toward "Mark" as it reads better as the title of the page and button text. Open to arguments.

Reply:
Good idea, changed to MarkActionPlanResourceAsCompleteViewModel/FormData